Rubberduck.Core/Formatters/InspectionResultFormatter.cs

```diff
@@ -0,0 +1,67 @@
+﻿using Rubberduck.Common;
+using Rubberduck.Parsing.Inspections.Abstract;
+using Rubberduck.Inspections.Abstract;
+using System.IO;
+using Rubberduck.Resources.Inspections;
+
+namespace Rubberduck.Formatters
+{
+    public class InspectionResultFormatter : IExportable
+    {
+        private readonly IInspectionResult _inspectionResult;
+
+        public InspectionResultFormatter(IInspectionResult inspectionResult)
+        {
+            _inspectionResult = inspectionResult;
+        }
+
+        public object[] ToArray()
+        {
+            var module = _inspectionResult.QualifiedSelection.QualifiedName;
+            return new object[]
+            {
+                _inspectionResult.Inspection.Severity.ToString(),
+                module.ProjectName,
+                module.ComponentName,
+                _inspectionResult.Description,
+                _inspectionResult.QualifiedSelection.Selection.StartLine,
+                _inspectionResult.QualifiedSelection.Selection.StartColumn
+            };
+        }
+
+        /// <summary>
+        /// WARNING: This property can have side effects. It can change the ActiveVBProject if the result has a null Declaration, 
+        /// which causes a flicker in the VBE. This should only be called if it is *absolutely* necessary.
+        /// </summary>
+        public string ToClipboardString()
+        {
+            var module = _inspectionResult.QualifiedSelection.QualifiedName;
+            var documentName = _inspectionResult.Target != null
+                ? _inspectionResult.Target.ProjectDisplayName
+                : string.Empty;
+
+            //todo: Find a sane way to reimplement this.
```

Comment:
Seems to me that we already did with the above code? IINM, they would give the same answer anyway in which case we can just chuck that commented-out code section.